src/notation/internal/notationuiactions.cpp

```diff
@@ -1569,7 +1569,7 @@ const UiActionList NotationUiActions::m_noteInputActions = {
     UiAction("note-longa",
              mu::context::UiCtxNotationOpened,
              QT_TRANSLATE_NOOP("action", "Longa"),
-             QT_TRANSLATE_NOOP("action", "Note duration: Longa"),
+             QT_TRANSLATE_NOOP("action", "Note duration: longa"),
```

Comment:
... this

Reply:
Also IMHO "Longa" is a "proper noun"?

Reply:
Wikipedia seems to spell it with a lowercase L when it's in the middle of a sentence https://en.wikipedia.org/wiki/Longa_(music)

Reply:
OK, so probably not a "proper noun", but still the first (and only) word after a colon

Reply:
I've done some digging and checked which of the groups with colons in the middle use an uppercase letter following the comma as of after this PR. Shall I make another PR to change the ones that aren't using an uppercase letter? (given that these ones seem to be in the minority)

Using uppercase:
- "Accessibility:"
- "Note input:"
- "Screen:"
- "Page:"
- "Toggle accidental:"

Not using uppercase
- "Grace:"
- "Note duration:"

- "Note input: rest" (I missed an individual string here, this goes against the rest of the group)